utils/install.py

```diff
@@ -64,10 +65,11 @@ def define_mongo_connection(context):
     context['mongo_user'] = ''
     context['mongo_password'] = ''
     if not test_mongodb_connection(db):
-        context['mongo_user'] = user_input("MongoDB username")
-        context['mongo_password'] = user_input("MongoDB password")
+            context['mongo_user'] = user_input("MongoDB user name")
+            context['mongo_password'] = user_input("MongoDB password")
+
         try:
-            db.authenticate(context['mongo_user'], quote_plus(context['mongo_password']))
+            db.authenticate(context['mongo_user'], context['mongo_password'])
```

Comment:
why remove `quote_plus` ?

Reply:
Quoting introduced a bug because the plus sign was encoded in the install script but not in the web interface and thus we could not loging with our admin user. Quoting the password also did not seem to be necessary at all. Please correct me, if that assumption is wrong.